draft/draft-ietf-bfd-optimizing-authentication.xml

```diff
@@ -392,12 +395,51 @@
         excepting that Auth Type is still TBD and that Reserved is set to 2.
       </t>
 
-      <section title="Error Handling">
+      <section title="Receipt Using Optimized Authentication">
+        <t>
+          The procedures for authenticating BFD Control packets is similar to
+          the existing procedures covered in <xref target="RFC5880" section="6.7"/>. 
+          The Optimized Authentication mode provides requirements for
+          authentication key processing and how the result of the authentication
+          operation is stored in the authentication section for that mode.
+          Similarly, the value of the Auth Len field is specified by the
+          Optimized Authentication mode in each of their specifications.
+        </t>
+        <t>
+          The following common procedures apply to authenticating BFD Control
+          packets utilizing Optimized Authentication:
+        </t>
+        <t>
+          If the received BFD Control packet does not contain an Authentication
```

Comment:
This is the first time we are referencing "Authentication Section" with capital letters (other than naming Figure 1). Otherwise, we seem to be referring to the section as "authentication section" in the rest of the document. Two things would help. Have a definition for the term in the Terminology section and be consistent with the use of the term with either the lower case or upper case letters.

Reply:
The capitalized form is from RFC 5880.  Part of next push cites that section and updates the capitalization elsewhere.